prime-router/docs/proposals/0009-proposal-for-history-api.md

```diff
@@ -0,0 +1,232 @@
+* PR [#1231](https://github.com/CDCgov/prime-reportstream/pull/1231) which adds report routing by trackingId further broken down by jurisdiction (federal, state and county). This information will be used for data in the end points.
+* Not a prerequisite, but useful to implement is proposal [#1285](https://github.com/CDCgov/prime-reportstream/issues/1285) which attempts to normalize messaging and summarize/aggregate to reduce the payload size. This may help fit the full response into the 2048 char `action_result` column with the full verbose response in the `action_result_json` column which will be used for end point querying.
+
+## Proposed End Points
+All endpoints will use the same security model as the `/settings/*` end points. Consideration should be given to check the user's role / Okta claims to ensure that they do have access to the organization to which the report is linked. Admins will be able to see all reports. Organization admins can see only their reports.
+
+### Report Submissions for a Given Date Range by Organization
+`GET /reports/{organization}?start=YYYY-MM-DD&end=YYYY-MM-DD&limit=20&offset=0`
+
+Parameters:
+* start - start date (inclusive) to query against the created_at date
+* end - end date (exclusive) to queery against the created_at date
+* limit - max number of submissions to pull back
+* offset - where to start the offset for pagination
+
+This end point will query the action and report_file tables to generate a list of all successful and failed `POST` submissions to the /api/reports endpoint. The intent is to show report submission frequency as well as capturing failed submissions. Pagination is supported with the limit and offset parameters. The resultSet provided in the response will allow for the UI to create pagination navigation elements.
+
+Example Json:
+```json
+{
+    "result_set": {
+        "count": 20,
+        "offset": 0,
+        "limit": 20,
+        "total": 3
+    },
+    "submissions": [
+        {
+            "actionId": 143,
+            "reportId" : "18a8e461-b3f8-436d-b94e-4065c7b9d186",
+            "createdAt": "2021-06-30 13:03:00.305657-04",
+            "reportItemCount" : 23,
+            "destinationCount" : 5,
+            "warningCount" : 8,
+            "errorCount" : 0,
+            "status": "CREATED"
+        },
+        {
+            "actionId": 158,
+            "reportId" : null,
+            "createdAt": "2021-06-30 13:03:00.305657-04",
+            "reportItemCount" : 0,
+            "destinationCount" : 0,
+            "warningCount" : 5,
+            "errorCount" : 3,
+            "status": "BAD_REQUEST"
+        },
+        {
+            "actionId": 177,
+            "reportId" : null,
+            "createdAt": "2021-06-30 13:03:00.305657-04",
+            "reportItemCount" : 0,
+            "destinationCount" : 0,
+            "warningCount" : 0,
+            "errorCount" : 0,
+            "status": "EXCEPTION"
```

Comment:
This will be the HttpStatus, true?  Or a text version of it.  Is there an http status 'exception'?